src/Tools/Source/RunTests/Options.cs

```diff
@@ -187,7 +187,7 @@ bool isOption(string argument, string optionName, out string value)
                     opt.ProcDumpDirectory = value;
                     index++;
                 }
-                else if (comparer.Equals(current, "-procdump"))
+                else if (comparer.Equals(current, "-useprocdump"))
                 {
                     opt.UseProcDump = false;
```

Comment:
Should this be `true`? ☕️

Reply:
That is fixed in https://github.com/dotnet/roslyn/pull/31472